### PR TITLE
Add a worker process.

### PR DESCRIPTION
**Pull request by ejholmes**
Running 1 process type is boring.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -14,12 +17,27 @@ func main() {
 	)
 	flag.Parse()
 
-	log.Printf("Starting on %s", *port)
-	log.Fatal(http.ListenAndServe(":"+*port, http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		log.Printf("%s - %s", r.Method, r.URL)
-		w.WriteHeader(200)
-		io.WriteString(w, "Ok\n")
-	})))
+	if len(os.Args) < 2 {
```
Comment (phobologic) [inline main.go]: use flag.Args here instead so you don't have to worry about the order quiet as much

File: main.go [hunk 2]
```diff
@@ -14,12 +17,27 @@ func main() {
...
+			w.WriteHeader(200)
+			io.WriteString(w, "Ok\n")
+		})))
+	case "worker":
+		for {
+			<-time.After(1 * time.Second)
+			fmt.Printf("Hard work %d...\n", rand.Int())
+		}
+	default:
+		log.Fatalf("Unkown subcommand: %s", cmd)
```
Comment (phobologic) [inline main.go]: Unknown

**Comment by phobologic** [comment]
:+1:

File: main.go (hunk 2 above)
Comment (ejholmes) [inline main.go]: God, I always manage to spell unknown wrong.

File: main.go (hunk 1 above)
Comment (ejholmes) [inline main.go]: Didn't know this. Nice!

**Pull request merged**